docs/dev/guidelines/code-review.rst

```diff
@@ -0,0 +1,61 @@
+***********
+Code Review
+***********
+
+STEPS
+======
+
+0. Precondition -> only Developer
+==================================
+
+* Limit yourself to one functionality per pull request
+* Split up your task in multiple branches & pull requests if necessary
+* `Commit Early, Commit Often, Perfect Later, Publish Once <https://speakerdeck.com/lemiorhan/10-git-anti-patterns-you-should-be-aware-of>`_
```

Comment:
Imho an article would be better than a slide show with mostly images, but I don't have one to suggest at hand.

Reply:
This is just based from the Confluence page, but yea we could improve this